app/models/payment/braintree_customer.rb

```diff
@@ -1,3 +1,8 @@
 class Payment::BraintreeCustomer < ActiveRecord::Base
   belongs_to :member
+  has_many   :payment_methods, class_name: Payment::BraintreePaymentMethod, foreign_key: 'customer_id'
```

Comment:
I double checked the docs, and `class_name` is always passed as a string, not the actual class. 
```